src/sentry/digests/utils.py

```diff
@@ -109,8 +122,10 @@ def build_custom_digest(original_digest: Digest, events: Iterable[Event]) -> Dig
             if user_group_records:
                 user_rule_groups[group] = user_group_records
         if user_rule_groups:
-            user_digest[rule] = user_rule_groups
+            if not skip:
```

Comment:
will skip ever be `True` here? if it is set to True in the inner for loop, won't it continue to the next iteration in the outer loop and reset skip to False?

Reply:
oh yeah woops reset it at the wrong spot